apps/framework-docs/src/pages/moose/building/ingestion.mdx

```diff
@@ -385,54 +332,6 @@ interface PipelineConfig<T> {
 }
 ```
 
-### Data Formats
```

Comment:
i couldn't figure out what this Data Formats section was documenting so i killed it. i'm assuming this is where the old batch option was documented?